stacks_queues/stack.py

```diff
@@ -21,18 +22,24 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store.head == None:
+            raise StackEmptyException("The stack is empty")
+
+        return self.store.remove_first()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        if self.store.head == None:
+            return True
+        else:
+            return False
```

Comment:
✨ This does work. You could also consider taking advantage of `LinkedList`'s `empty` method